docs/source/guides/ursula_configuration_guide.rst

```diff
@@ -23,11 +20,26 @@ If your installation in non-functional, be sure you have the latest version inst
 .. _Installation Guide: installation_guide.html
 
 
-2. Configure a new Ursula node
 
+2(a). Configure a new Ursula node
+
+*Decentralized Ursula Configuration*
+.. code:: bash
+
+    (nucypher)$ nucypher ursula init --provider-uri <YOUR PROVIDER URI> --network <NETWORK NAME>
```

Comment:
```suggestion
    (nucypher)$ nucypher ursula init --poa --provider-uri <YOUR PROVIDER URI> --network <NETWORK NAME>
```

Reply:
Ah, nice catch.  Yeah this can live here until we officially support other nodes.

Reply:
I made a note below explaining when to use --poa :heavy_check_mark: